### PR TITLE
Dbaas 2120

### PR DESCRIPTION
**Pull request by Ben-Epstein**

### CODE REVIEW

**Comment by njnygaard** [comment]
What is the pathway for getting these in new clusters? Is master pulled for all new clusters?

**Comment by Ben-Epstein** [comment]
@njnygaard No, we will create a new release and update dcos-commons-vanilla to pull from the new one

**Review by njnygaard — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -45,8 +44,13 @@ You can add notebooks by making Github pull request, but we request you also upd
 |2CG6KVZJ2|5. Reference Applications / 2. IoT / 5. Splice Query| |[json](https://github.com/splicemachine/zeppelin-notebooks/raw/master/2CG6KVZJ2/note.json) | |
 |2D2RM85JF|6. Advanced Tutorials / 1. Creating Custom Stored Procedures| |[json](https://github.com/splicemachine/zeppelin-notebooks/raw/master/2D2RM85JF/note.json) | |
 |2CSGDX1CW|7. Benchmarks / 1. TPCH-100| |[json](https://github.com/splicemachine/zeppelin-notebooks/raw/master/2CSGDX1CW/note.json) | |
+|2CN9CP75G|8. Machine Learning / 1. Kmeans| |[json](https://github.com/splicemachine/zeppelin-notebooks/raw/master/2CN9CP75G/note.json) | |
```
Comment (edriggers) [inline README.md]: I don't see any of these '8. Machine Learning' notebooks

**Review by edriggers — CHANGES_REQUESTED** [review]
A couple of comments that are in general to all notebooks:
1 - Clear the output before checking in notebook for paragraphs that are not 'md' paragraphs
2 - Format the JSON before checking in the file - In IntelliJ the command is Command Option L
3 - If you are making changes to existing notebooks please check them in with the same file names.  I know the name of the file changes in some scenarios but you can update the JSON and change the id (normally near the bottom of the json) to match the original value.  The reason for this is that it is hard to see what changed between the version you are checking in and the original version.
4 - If we are getting rid of the Data Engineering folder then we need to fix the numbering of the top level folders.

**Comment by Ben-Epstein** [comment]
@edriggers  Regarding 3, I understand and will keep with that protocol from now on, but I don't think it's worth changing them for this PR, because it would require going into every notebook to change back which URL the previous notebook links to because each notebook in the series references the one after it.

**Review by edriggers — APPROVED** [review]

**Pull request merged**